src/components/Users/UsersContainer.js

```diff
@@ -1,38 +1,88 @@
+import React from 'react';
 import { connect } from 'react-redux';
 
+import Preloader from '../../utils/preloader';
+
+import axios from 'axios';
+
 import {
-  followActionCreator,
-  unfollowActionCreator,
-  setUsersActionCreator,
-  setCurrentPageActionCreator,
-  setTotalUsersCountActionCreator,
+    followActionCreator,
+    unfollowActionCreator,
+    setUsersActionCreator,
+    setCurrentPageActionCreator,
+    setTotalUsersCountActionCreator,
+    toggleIsFetchingActionCreator
 } from '../../redux/actions/actionCreators';
 
-import UsersAPIComponent from './UsersAPIComponent';
+import Users from './Users';
+
+class UsersContainer extends React.Component {
+
+    componentDidMount() {
+        this.props.toggleIsFetching(true);
+        axios.get(`https://social-network.samuraijs.com/api/1.0/users?page=${this.props.currentPage}&count=${this.props.pageSize}`)
```

Comment:
Consider storing the api url in the config directory of your core folder.

Reply:
I thought about it right away, but I didn’t succeed, the project wasn’t started up to because it was a pattern string, which needed props, which by itself are not in the utilities.

Reply:
You can just store "https://social-network.samuraijs.com/api/1.0/users" and then concat the rest of the dynamic parameters.

Reply:
/fxd